tests/h/emails/reply_notification_test.py

```diff
@@ -1,23 +1,14 @@
 import datetime
-from unittest import mock
 
 import pytest
 
 from h.emails.reply_notification import generate
-from h.models import Annotation, Document
+from h.models import Annotation, Document, Subscriptions
 from h.notification.reply import Notification
 
```

Comment:
These tests are something else, as is the code they test with about 20 conditionals. I've tried to be relatively surgical and leave most of this in place.

Hoo boy there's room for improvement, but we also have other priorities.